### DISCUSSION TITLE
Custom font not detected on iOS

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [skiptools/skip#283] [Q&A] [answered]
Using SkipUI 1.13.10, custom fonts are not detected on the iOS side when following the documentation. They do work on Android tho.

<img width="1468" alt="image" src="https://github.com/user-attachments/assets/7496caa7-a90f-4316-9aa5-a87474217251">

I end up with this error which explains itself:

```
FontParser could not open filePath /Users/sinan/Library/Developer/CoreSimulator/Devices/1931C460-0C56-4385-8114-0963A98C5038/data/Containers/Bundle/Application/FA1FAD64-0CF5-4F3F-A9F9-7AAF17EEA86C/Localz.app/comfortaa.tff: [2: No such file or directory]
```

I did followed the documentation steps, and I also compared side by side with the Showcase app but couldn't find any difference. Thanks!

**Comment by marcprux** [comment, marked as answer]
I think you misspelled "comfortaa.**ttf**" as "comfortaa.**tff**" in your `Info.plist`…

**Reply by Sinan-Karakaya** [reply]
Well that one is embarassing... Thanks still
